plugins/module_utils/client/resource.py

```diff
@@ -0,0 +1,49 @@
+# Copyright [2017] [Red Hat, Inc.]
```

Comment:
2021?

Reply:
To be honest, no idea. I'm going off what I think @Akasurde said in earlier PRs for this client migration work.